app/assets/javascripts/wizard/components/theme-preview.js.es6

```diff
@@ -46,41 +46,44 @@ export default createPreviewComponent(659, 320, {
     ctx.beginPath();
     ctx.fillStyle = colors.primary;
     ctx.font = `bold ${titleFontSize}em 'Arial'`;
-    ctx.fillText("Welcome to Discourse", margin, height * 0.25);
+    ctx.fillText("Discussion topic", margin, height * 0.3);
 
-    const bodyFontSize = height / 440.0;
+    const bodyFontSize = height / 220.0;
     ctx.font = `${bodyFontSize}em 'Arial'`;
 
     let line = 0;
     const lines = LOREM.split("\n");
-    for (let i = 0; i < 10; i++) {
-      line = height * 0.3 + i * lineHeight;
+    for (let i = 0; i < 4; i++) {
+      line = height * 0.35 + i * lineHeight;
       ctx.fillText(lines[i], margin + avatarSize + margin, line);
     }
 
+    // Share Button
+    ctx.beginPath();
+    ctx.rect(margin, line + lineHeight, width * 0.14, height * 0.14);
+    ctx.fillStyle = darkLightDiff(colors.primary, colors.secondary, 90, 65);
+    ctx.fill();
+    ctx.fillStyle = chooseDarker(colors.primary, colors.secondary);
+    ctx.font = `${bodyFontSize}em 'Arial'`;
+    ctx.fillText("Share", margin + width/55, line + lineHeight * 1.85);
```

Comment:
We use prettier now, and this line should be formatted like this:

```
ctx.fillText("Share", margin + width / 55, line + lineHeight * 1.85);
```

I recommend you setup some prettier integration in your editor which will format on save, there are various prettier offenses in this PR. Let me know if you need help.

https://prettier.io/